openpype/pipeline/file_collection.py

```diff
@@ -0,0 +1,274 @@
+"""Utility functions for file sequences and collections
+"""
+import os
+import re
+import clique
+from openpype.lib import Logger
+from openpype.pipeline import frames
+
+
+def collect_filepaths_from_sequential_path(
+    file_path,
+    frame_start=None,
+    frame_end=None,
+    only_existing=False,
+):
+    """Generate expected files from path
```

Comment:
Can we add some example `path` to the docstring?

```python
collect_filepaths_from_sequential_path("/path/to/fname.####.exr")
collect_filepaths_from_sequential_path("/path/to/fname.%d.exr")
collect_filepaths_from_sequential_path("name.%04d.exr")
collect_filepaths_from_sequential_path("name.<frame>.exr")
collect_filepaths_from_sequential_path("name.[1001-1050].exr")
```

It's not that clear from the docstring that:
- The paths must be absolute paths, not filenames
- That only #### and %04d are supported

Reply:
more info here 953e1d8fb48ee0ce21ca4c6bb6c8ea35a52a8846